mlbench_core/cli/cli.py

```diff
@@ -284,7 +284,7 @@ def delete(name, dashboard_url):
 
 @cli.command()
 @click.argument("name", type=str)
-@click.option("--output", "-o", type=str)
+@click.option("--output", "-o", default="results", type=str)
```

Comment:
Isn't the output file a zip archive? "results.zip" would be less confusing. :)

Reply:
I agree. I fixed it in the new commit. Thanks!